# Require data-testid attribute on form elements (require-testid)

Intended for enforcing reliable selectors to clickable elements


## Rule Details

Examples of **incorrect** code for this rule:

```js

<button></button>

```

Examples of **correct** code for this rule:

```js

<button data-testid="my-form-button"></selbuttonect>

```
